projects/netgrif-components-core/src/lib/search/models/category/case/case-title.ts

```diff
@@ -38,4 +38,8 @@ export class CaseTitle extends NoConfigurationCategory<string> {
     serializeClass(): Categories | string {
         return Categories.CASE_TITLE;
     }
+
+    protected get elasticKeywords(): Array<string> {
+        return [`${CaseSearch.TITLE}${this.isSelectedOperator(Substring) ? '.keyword' : ''}`];
```

Comment:
Since a similar method was introduced into multiple classes, it might be better to add a generalization to the `SearchIndexResolverService` and use this dependency in all categories

Reply:
Done